.changeset/hot-hornets-laugh.md

```diff
@@ -0,0 +1,12 @@
+---
+'@backstage/plugin-search-backend-module-elasticsearch': minor
+'@backstage/plugin-search-backend-node': minor
+'@backstage/plugin-search-common': minor
+'@backstage/plugin-search-react': minor
+---
```

Comment:
I'd recommend splitting this up into possibly two or three distinct changesets, just to keep the changelog entries focused for each package.  For example:

- One could be just `@backstage/plugin-search-common` that mentions that it's now possible to provide and consume a total number of results on a search result response.
- One could be just `@backstage/plugin-search-react` that mentions that the `value` of a `search` analytics event is now set as the total number of results (when available)
- And then you could write one shared across the various search backends that just mentions that a number of results is now provided alongside the query result.

Reply:
Thanks - done